gradle.properties

```diff
@@ -1,11 +1,11 @@
 # Project
-version=2.2
-version_patch=1
+version=2.3
+version_patch=0
```

Comment:
Just update the patch version to .2. The overall mod version is still 2.2. This is simply a port and not a feature upgrade :)

So the version will be 2.2.2

Reply:
Makes sense to me, updated it